conda_forge_tick/migrators.xsh

```diff
@@ -27,7 +29,11 @@ class Migrator:
             True if node is to be skipped
         """
         # never run on archived feedstocks
-        return bool(attrs.get('archived', False))
+        # don't run on things we've already done
+        # don't run on bad nodes
+        return (bool(attrs.get('archived', False)
+                or self.migrator_hash(attrs) in attrs.get('PRed', []))
+                or attrs.get('bad', False))
```

Comment:
The problem with this is that the graph thinks that good nodes that used to be bad are still bad. We would need to set bad to False somewhere earlier in the code.

Reply:
Hmm ok, should we reset bad here? https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/make_graph.py#L15
That way every node is good at least to start with.
Or would this introduce too much of a burden?

Reply:
I think that should be fine.

Reply:
We may need to think about this more, since failing on every bad feedstock every time the bot runs could be a problem.